app/views/bright_events/index.html.erb

```diff
@@ -0,0 +1,50 @@
+<div>
+  <div>
+  <%= link_to bright_events_path , class: 'btn btn-add' do %>
+    <i class="uil uil-plus"></i>
+    <span>Agregar evento</span>
+  <% end %>
+    
+  </div>
+  <div class="contentTable">
+    <table>
+      <tr>
+        <th>Título</th>
+        <th>Descripción</th>
+        <th>Fecha</th>
+        <th>Ubicación</th>
+        <th>Editar</th>
+        <th>Ver</th>
+        <th>Eliminar</th>
+      </tr>
```

Comment:
if you're using local storage data through i18n, you can retrieve this kind of data from an es.yml file like this:

```
es:
  titles:
    title: 'Título'
    description: 'Descripción'
    date: 'Fecha'
    location: 'Ubicación'
    edit: 'Editar'
    view: 'Ver'
    delete: 'Eliminar'
```

With this approach, you will now be able to do something like this in your views:
```
 <tr>
    <% t('.titles').each do |key, value| %>
      <th><%= value %></th>
    <% end %>
  </tr>
```